app/views/logs/delete_lettings_logs.html.erb

```diff
@@ -0,0 +1,49 @@
+<% title = "Delete logs" %>
+<% content_for :title, title %>
+<% content_for :before_content do %>
+  <%= govuk_back_link(href: :back) %>
+<% end %>
+
+<h1 class="govuk-heading-l">
+  <span class="govuk-caption-l"><%= title %></span>
+  Review the logs you want to delete
+</h1>
+<% logs = @delete_logs_form.logs %>
+<p>You've selected <%= logs&.count %> <%= "log".pluralize(logs&.count) %> to delete</p>
```

Comment:
the form can act a view object so i wouldn't bother with the local variable and just reference `@delete_logs_form` wholly instead

as an example if you want less code is to push stuff/logic downwards so the responsibility becomes delegated. if the form implemented `form.log_count` you could push that downwards then view does not need to deal with nil guards. the view just wants to ask for some info ie log count to render. it shouldn't need to deal with nil guards as that is someone else's responsibility, in this case the form/view.object